tests/test_bandpass.py

```diff
@@ -276,9 +276,26 @@ def test_ne():
     t2 = time.time()
     print 'time for %s = %.2f' % (funcname(), t2-t1)
 
+
+def test_thin():
```

Comment:
Should we also do the test with a thinned SED and bandpass (for some reasonable SED)?  I would think so.